internal/version/VERSION

```diff
@@ -1 +1 @@
-0.2.4
+0.2.5
```

Comment:
can you upgrade version in [example](https://github.com/spiffe/spiffe-csi/blob/main/example/config/spiffe-csi-driver.yaml#L32)?

Reply:
Good catch! I've updated it in 609b851.